github_connector/demo/github_analysis_rule_demo.xml

```diff
@@ -1,23 +1,35 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!--
-Copyright 2020 Tecnativa - Víctor Martínez
+Copyright 2020-2023 Tecnativa - Víctor Martínez
 License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 <odoo>
     <record id="github_analysis_rule_python" model="github.analysis.rule">
         <field name="name">Python files</field>
         <field name="group_id" ref="github_analysis_rule_group_1_demo" />
-        <field name="paths">*.py</field>
+        <field name="paths">
+            *.py
+            !/setup
+            !.*
```

Comment:
what this pattern is excluding ?

Reply:
This is intended to exclude unnecessary files for analysis (.pre-commit-config.yaml, .travis.yaml, etc).

Reply:
OK ! Thanks !